test/ai.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { before, describe, it } from "mocha";
+import { OPENAI_ERROR, fetchUrlSummary } from "../bots/ai.js";
+import { expect } from "chai";
+
+describe("Fetching URL Summary", () => {
```

Comment:
need to add a test case for the successful case where a description is returned. to do this you will need to mock the `OpenAi` client. I'd suggest updating the `fetchUrlSummary` function to take a 3rd argument which is an optional `OpenAi` client that is only passed in the tests.